README.md

```diff
@@ -1,17 +1,23 @@
 # cloudland2023-otel-workshop
 
-A tutorial for monitoring and troubleshooting applications in production using OpenTelemetry.
+_Workshop content for a session during
+[Cloud Land 2023](https://www.cloudland.org/en/home/) -- The content may not be
+updated after the conference_
```

Comment:
Copied that from my workshop repo, I think it is good to let people know who might find this repo AFTER the conference that this has been the purpose of it

Reply:
good idea. thanks!